pkg/config/config.go

```diff
@@ -606,7 +606,7 @@ func initConfig(config Config) {
 	config.SetKnown("process_config.expvar_port")
 
 	// System probe
-	config.SetKnown("system_probe_config.enabled")
+	config.BindEnv("system_probe_config.enabled")
```

Comment:
why did this change?

Reply:
Sorry! I'll change this back - it was testing something about viper and I forgot to change it back. Thank you for catching it!